Update 2025-reed-systems.qmd

erblum opened the pull request:
Thanks for moving so quickly on our pull request! I spotted one typo ('Oregan' should be 'Oregon'); I hope making this pull request is a good way to go about fixing it.

annaritz commented:
Hi @shilad - the current list is for tenure-track positions only, unfortunately, I don't think this website was intended to also list visiting positions (@evanpeck correct me if I'm wrong). Thanks - Anna

evanpeck commented:
Just closing this since it's out of date. FYI: we'll be adding visiting positions this year, so if another one comes around, feel free to add it.

erblum committed:
Update 2025-reed-systems.qmd

Thanks for moving so quickly on our pull request! I spotted one typo ('Oregan' should be 'Oregon'); I hope making this pull request is a good way to go about fixing it.

PhilChodrow commented:
It's great, thank you! All best for the search.
